datadog/measure.go

```diff
@@ -9,7 +9,7 @@ import (
 
 // AppendMeasure is a formatting routine to append the dogstatsd protocol
 // representation of a measure to a memory buffer.
-func AppendMeasure(b []byte, m stats.Measure) []byte {
+func AppendMeasure(b []byte, m stats.Measure, filters map[string]struct{}) []byte {
```

Comment:
To avoid the breaking change here, do you think we could keep this function the way it was an introduce a new one instead?
```go
func AppendMeasure(b []byte, m stats.Measure) []byte {
    return appendMeasureFiltered(b, m, nil)
}

func appendMeasureFiltered(b []byte, m stats.Measure, filters map[string]struct{}) []byte {
    // ...
}
```

Reply:
Ya definitely. A quick search of the repo gave me the impression it was only used in the one spot, which is why I went for the change.

Reply:
Yes, it's very unlikely that there is code out there that depends on this function, but since it's exported we have no way to know.